tests/infra/proposal_generator.py

```diff
@@ -0,0 +1,334 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import argparse
+import inspect
+import json
+import os
+
+from loguru import logger as LOG
+
+
+def dump_proposal(output_path, proposal, dump_args):
+    LOG.info(f"Writing proposal to {output_path}")
+    with open(output_path, "w") as f:
+        json.dump(proposal, f, **dump_args)
+
+
+def dump_vote(output_path, vote, dump_args):
+    LOG.info(f"Writing vote to {output_path}")
+    with open(output_path, "w") as f:
+        json.dump(vote, f, **dump_args)
+
+
+def file_to_byte_array(path):
```

Comment:
If we're working on usability/friendliness of the governance API, is it time that we move those to base64 strings?

Reply:
Absolutely - looking at this in a separate PR

Reply:
For posterity, that's #1374.